### PR TITLE
chore: update viem, token lists and fix chains in account history

### PR DESCRIPTION
**Pull request by 0xArdy**
closes SUP-83

### CODE REVIEW

**Review by 0xtiti — APPROVED** [review]
lgtm!

File: src/types/data.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ export interface CustomClients {
     contracts: OpContracts; // contracts for the from chain
   };
   to: {
-    wallet: WalletClient & (WalletActionsL1 & WalletActionsL2);
+    wallet?: WalletClient & (WalletActionsL1 & WalletActionsL2);
```
Comment (turtlemoji) [inline src/types/data.ts:19]: wallet should be optional?

File: src/utils/variables.ts [hunk 2]
```diff
@@ -31,10 +80,15 @@ export const chainData: ChainData = {
...
+  [zoraSepolia.id]: {
     logo: baseSepoliaLogo.src,
-    apiUrl: 'https://api.basescan.org/api',
-    explorer: 'https://basescan.org//',
+    apiUrl: '',
+    explorer: '',
+  },
+  [liskSepolia.id]: {
+    logo: liskLogo.src,
+    apiUrl: '',
```
Comment (turtlemoji) [inline src/utils/variables.ts:90]: TODOS?

**Review by turtlemoji — DISMISSED** [review]
pre-approved with comments

File: src/types/data.ts (hunk 1 above)
Comment (0xArdy) [reply src/types/data.ts:19]: yes, `walletClient` should be `undefined` if the user is disconnected

File: src/utils/variables.ts (hunk 2 above)
Comment (0xArdy) [reply src/utils/variables.ts:90]: yep, i create a linear task for that https://linear.app/defi-wonderland/issue/SUP-89/fill-chain-explorers-in-variablests-file

**Review by 0xtiti — APPROVED** [review]

**Pull request merged**